### ISSUE TITLE
Makefile CI Build Failure (make test)

### ISSUE THREAD

**Issue by 0x4248** [Proactive-Development/pstat#2]
Build [#10](https://github.com/Proactive-Development/pstat/actions/runs/2018125682) Fails on make test
___
```bash
zip -r app_build.zip *
updating: __main__.py (deflated 77%)
make: pstat: Command not found
make: *** [makefile:17: test] Error 127
updating: makefile (deflated 62%)
  adding: pstat (deflated [5](https://github.com/Proactive-Development/pstat/runs/5632941243?check_suite_focus=true#step:4:5)%)
echo '#!/usr/bin/env python3' | cat - app_build.zip > pstat
chmod +x pstat
pstat -V 

```
**Error: Process completed with exit code 2.**
___

**Comment by 0x4248** [comment]
Its because it is not in the `usr/local/bin` folder.
Running `./pstat` works

**Comment by 0x4248** [comment]
Fixed [#14](https://github.com/Proactive-Development/pstat/runs/5633490892)
